var/spack/repos/builtin/packages/chapel/package.py

```diff
@@ -489,6 +504,13 @@ class Chapel(AutotoolsPackage, CudaPackage, ROCmPackage):
         "https://chapel-lang.org/docs/usingchapel/chplenv.html#chpl-host-jemalloc",
     )
 
+    conflicts(
+        "llvm=none",
+        when="+python-bindings",
+        msg="Python bindings require building with LLVM, see "
+        "https://chapel-lang.org/docs/tools/chapel-py/chapel-py.html#installation",
```

Comment:
I suggest this instead be added directly below beside the same existing conflicts for CUDA or ROCM with `llvm=none`